docs_src/security/Ch-SecretStore.md

```diff
@@ -1134,6 +290,5 @@ man-style documentation:
 
 -   [security-file-token-provider](./security-file-token-provider.1.md) -
     Generate Vault tokens for EdgeX services
--   [security-secrets-setup](./security-secrets-setup.1.md) - Creates an
-    on-device public-key infrastructure (PKI) to secure microservice
-    secret management
+-   [secrets-config](./secrets-config.1.md) - Utility for secrets management.
```

Comment:
`.1.md` used again.... I understand that is old school Linux man page syntax, but these are not man pages. Please use standard file markdown file naming.